src/battle_script_commands.c

```diff
@@ -3537,12 +3767,29 @@ static void Cmd_getexp(void)
                     else
                         gBattleMoveDamage = 0;
 
+                    if (gBattleTypeFlags & BATTLE_TYPE_TRAINER && B_TRAINER_EXP_MULTIPLIER != GEN_7)
```

Comment:
this shouldn't be moved. Can you put it back where it was before and

Reply:
Kept it above exp scaling since the trainer multiplier goes before that, but otherwise cleaned up the double trainer check